Model/Method/Ideal.php

```diff
@@ -92,6 +87,21 @@ public function getOrderTransactionBuilder($payment)
         return $transactionBuilder;
     }
 
+    private function getOrderRequestParameters($payment): array
+    {
+        $parameters = [];
+
+        /** @var IdealConfig $config */
+        $config = $this->objectManager->get(IdealConfig::class);
```

Comment:
You can use getConfigData method to get show_issuers, instead of using objectManger